app/views/recipes/public.html.erb

```diff
@@ -0,0 +1,14 @@
+<section class='container'>
+  <% @recipes.each do |recipe| %>
+    <a href="<%= recipe_path(recipe.id)%>" class='card'>
+      <div class='subdiv'>
+        <h2><%= recipe.name %></h2>
+        <h3>By: <%= recipe.user.name %></h3>
+      </div>
+      <div class='subdiv'>
+        <p>Total food items: <%= recipe.total_items %></p>
+        <p>Total price: $<%= recipe.total_price %></p>
+      </div>
+    </a>
```

Comment:
- [ ] - [ ] @asdt560 and @d4nQw3rty good work on your views however it's best practice to use partial views for your recipes, kindly add it as you did for the foods.